lib/utils/determine_icon.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:collection/collection.dart';
+
+import '../config/ui_config.dart';
+
+extension IconDeterminer on String? {
+  String determineIcon() {
+    const icons = {
```

Comment:
Maybe this map should go to separate config file in `/config` dir?

Reply:
Good Morning :-) 
Good point, why not.